src/verifiers/signedProof/openAttestationSignedProof.test.ts

```diff
@@ -0,0 +1,58 @@
+import { openAttestationSignedProof } from "./openAttestationSignedProof";
```

Comment:
Could you slightly change the way of testing like https://github.com/Open-Attestation/oa-verify/blob/master/src/verifiers/dnsText/openAttestationDnsTxt.v2.test.ts

1. build a verifier with this verification method
1. run the verifier

=> there is no need to call .test or .verify method of the verification method, just call the verifier and make sure the output is as expected
=> let's keep all the current use cases

Reply:
much nicer!